linode/networkingipassignment/framework_resource_schema.go

```diff
@@ -0,0 +1,38 @@
+package networkingipassignment
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/listplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+var frameworkResourceSchema = schema.Schema{
+	Attributes: map[string]schema.Attribute{
+		"id": schema.StringAttribute{
+			Computed:    true,
+			Description: "The ID of the IP assignment operation.",
+			PlanModifiers: []planmodifier.String{
+				stringplanmodifier.UseStateForUnknown(), // Use the state when ID is unknown.
+			},
+		},
+		"region": schema.StringAttribute{
+			Required:    true,
+			Description: "The region for the IP assignments.",
+		},
+		"assignments": schema.ListAttribute{
```

Comment:
Could we use a SetAttribute here to make arbitrary order changes not cause a diff?